metadata/PerLanguageSummaries/VerbInflectionMacrocategoriesAggregatedByMarkerPosition.yaml

```diff
@@ -41,82 +41,207 @@ fields:
+      "post": >-
+        after the phonological host
+      "in/post": >-
+        either inside or after the phonological host, conditioned by some structural
+        context (category, syntax etc.)
   # -- end of MarkerPositionForInterClausal
 
   MarkerPositionForOther:
     description: >-
       GrammaticalMarkers::MarkerPosition value for Other
     kind: computed data (aggregation-scripts/VerbInflectionPerLanguage.R)
-    data: logical
+    data: value-list
+    values: 
+      "prae": >-
+        before the phonological host
   # -- end of MarkerPositionForOther
 
   MarkerPositionForPragmatic:
     description: >-
       GrammaticalMarkers::MarkerPosition value for Pragmatic
     kind: computed data (aggregation-scripts/VerbInflectionPerLanguage.R)
-    data: logical
+    data: value-list
+    values: 
+      "prae": >-
+        before the phonological host
+      "in": >-
+        inside the phonological host
+      "post": >-
+        after the phonological host
+      "in (post)": >-
+        inside the phonological host by default, after the host under exceptional
+        conditions (typically lexical)
+      "multiple": >-
+        multiple markers at different positions
   # -- end of MarkerPositionForPragmatic
 
   MarkerPositionForNumber:
     description: >-
       GrammaticalMarkers::MarkerPosition value for Number
     kind: computed data (aggregation-scripts/VerbInflectionPerLanguage.R)
-    data: logical
+    data: value-list
+    values: 
+      "prae": >-
+        before the phonological host
+      "in": >-
+        inside the phonological host
+      "post": >-
+        after the phonological host
+      "in/post": >-
+        either inside or after the phonological host, conditioned by some structural
+        context (category, syntax etc.)
+      "multiple": >-
+        multiple markers at different positions
   # -- end of MarkerPositionForNumber
 
   MarkerPositionForEvidential:
     description: >-
       GrammaticalMarkers::MarkerPosition value for Evidential
     kind: computed data (aggregation-scripts/VerbInflectionPerLanguage.R)
-    data: logical
+    data: value-list
+    values: 
+      "post": >-
+        after the phonological host
   # -- end of MarkerPositionForEvidential
 
   MarkerPositionForNPRelated:
     description: >-
       GrammaticalMarkers::MarkerPosition value for NPRelated
     kind: computed data (aggregation-scripts/VerbInflectionPerLanguage.R)
-    data: logical
+    data: value-list
+    values: 
+      "prae": >-
+        before the phonological host
+      "in": >-
+        inside the phonological host
+      "post": >-
+        after the phonological host
   # -- end of MarkerPositionForNPRelated
 
   MarkerPositionForClassification:
     description: >-
       GrammaticalMarkers::MarkerPosition value for Classification
     kind: computed data (aggregation-scripts/VerbInflectionPerLanguage.R)
-    data: logical
+    data: value-list
+    values: []
```

Comment:
This is supposed to be a dictionary, not a list.

Reply:
There's a couple more like this.

Reply:
Good point. Fixed in c52d68af4646034adc58e32875f62d1669bbcf9c